teamsupport/models.py

```diff
@@ -1,19 +1,31 @@
 from property_caching import cached_property
 from querylist import QueryList
 
+from teamsupport.services import TeamSupportService
 from teamsupport.errors import MissingArgumentError
 
 
+TICKET_STATUS_NEW = '212203'
+TICKET_TYPE_SUPPORT = '35731'
```

Comment:
Same values (I hope) because they are primary keys from TicketStatuses/TicketType tables. I can't believe they can change. Regarding getting them dynamically - how to find them? Looking for "New" string? If's far less reliable than primary keys as for me.